scripts/gen_thumbnails.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+STATIC_DIR=../static/samples/
+THUMB_WIDTH=200
+THUMB_HEIGHT=10000 # height: auto;
+
+rm -f $STATIC_DIR*-thumb.jpg
+
+for f in $STATIC_DIR*.jpg
```

Comment:
Better:

> for f in "$STATIC_DIR"*.jpg

I guess the same for `rm -f`
Source:
```
$ shellcheck myscript
 
Line 11:
for f in $STATIC_DIR*.jpg
         ^-- SC2231: Quote expansions in this for loop glob to prevent wordsplitting, e.g. "$dir"/*.txt .

Reply:
Fixed it, thanks. For me, `shellcheck` now returns without any complaints.